spec/Types.tex

```diff
@@ -249,6 +250,21 @@ \subsection{The String Type}
 sets, possibly including internationalization.
 \end{openissue}
 
+\subsection{The Bytes Type}
+\label{The_Bytes_Type}
+\index{bytes@\chpl{bytes}}
+\index{types!bytes@\chpl{bytes}}
+
+Bytes is a primitive type designated by the symbol \chpl{bytes} comprised of
+arbitrary bytes. Methods on bytes that interpret the data as characters assume
+that the bytes are ASCII characters.
+    
+Bytes are immutable in-place and their length is unbounded.
+
```

Comment:
Let us remove one of the two empty lines here, and the same in the Strings subsection above.